tests/e2e/support/objects/app-files/share/actions.ts

```diff
@@ -25,11 +25,14 @@ const pendingShareItem =
   '//div[@id="files-shared-with-me-pending-section"]//tr[contains(@class,"oc-tbody-tr")]'
 const passwordInput = '.oc-modal-body input.oc-text-input'
 const createLinkButton = '.oc-modal-body-actions-confirm'
+const showMoreOptionsButton = '#show-more-share-options-btn'
+const calendarDatePickerId = 'recipient-datepicker-btn'
```

Comment:
```suggestion
const calendarDatePickerId = '#recipient-datepicker-btn'
```

Reply:
I don't need # because I'm using `page.getByTestId()`. https://playwright.dev/docs/api/class-page#page-get-by-test-id

Reply:
Ahh, so is it `data-testid`?

Reply:
yeah
<img width="1105" alt="Screenshot 2024-01-05 at 13 01 39" src="https://github.com/owncloud/web/assets/84779829/6775509d-be64-420b-860a-98f3eb0486d2">